core-backend/sandbox/src/runtime.rs

```diff
@@ -47,6 +47,9 @@ where
     }
 
     fn read_memory(&self, ptr: u32, len: u32) -> Result<Vec<u8>, MemoryError> {
+        if len as usize > RUNTIME_MAX_ALLOC_SIZE {
```

Comment:
should we provide same check for write output?
and propagate some other error instead of out of bounds to identify it in future easier

Reply:
This problem must be checked in place where buffer is allocated, not in place where it uses. So, currently I cannot find any place for which this problem can occurs where `write_output` is used.

Reply:
Talking about different error - is a good question. May be we really should make new error. But currently this error can occurs only in case where wasm memory bound error will also occurs, so I suppose leave one error for both cases is not criminal now. I will make issue: we must investigate all runtime and check that over size buffers cannot be created anywhere and also this question will be there